packages/react-focus/src/components/FocusZone/FocusZone.test.tsx

```diff
@@ -91,7 +91,10 @@ describe('FocusZone', () => {
     elementRefName: 'elementRef',
     testOptions: {
       'consistent-callback-names': { ignoreProps: ['onActiveElementChanged'] },
-      'consistent-callback-args': {
+      'consistent-callback-args-legacy': {
+        ignoreProps: ['onActiveElementChanged', 'onBeforeFocus', 'onFocusNotification', 'onFocus'],
+      },
+      'consistent-callback-type': {
```

Comment:
why is this being added to v8 package ?

Reply:
Because we want to disable the new `consistent-callback-type` for v8. 
If you meant `consistent-callback-args-legacy`, the git diff is not very smart there, it was already in v8.

Reply:
no I meant the new callback-type. 

it should be disabled in root `isConformant` definition rather than configured for ignoring specific handlers
<img width="904" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/26febaad-ce99-4261-9eab-41b51efc4dea">